consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/StubbedSynchronizerUpdater.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.ibft.support;
+
+import tech.pegasys.pantheon.consensus.ibft.BaseSynchronizerUpdater;
+import tech.pegasys.pantheon.ethereum.p2p.api.PeerConnection;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class StubbedSynchronizerUpdater implements BaseSynchronizerUpdater {
+
+  private Map<PeerConnection, ValidatorPeer> validatorNodes;
```

Comment:
What am I missing here, doesn't validatorNodes need initialising to avoid NPE?

Reply:
There's no NPE as the addNetworkPeers creates the map and assigns the created map to this variable.

Reply:
👍